scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -918,6 +924,428 @@ cdef class TransformedDensityRejection(Method):
+    percentile corressponding to the quantile ``U`` i.e. ``X = approx_ppf(U)``. We call
+    the maximal tolerated u-error the u-resolution of the algorithm in the sequel.
+
+    Both the order of the interpolating polynomial and the u-resolution can be selected.
+    Note that very small values of the u-resolution are possible but increase the cost
+    for the setup step.
+
+    The interpolating polynomials have to be computed in a setup step. However, it only
+    works for distributions with bounded domain; for distributions with unbounded domain
+    the tails are cut off such that the probability for the tail regions is small compared
+    to the given u-resolution.
+
+    The construction of the interpolation polynomial only works when the PDF is unimodal
+    or when the PDF does not vanish between two modes.
+
+    Parameters
+    ----------
+    dist : object
+        An instance of a class with a ``pdf`` and optionally a ``cdf`` method.
+
+        * ``pdf``: PDF of the distribution. The signature of the PDF is expected to be:
+          ``def pdf(self, x: float) -> float``. i.e. the PDF should accept a Python
+          float and return a Python float. It doesn't need to integrate to 1 i.e. the
+          PDF doesn't need to be normalized.
+        * ``cdf``: CDF of the distribution. This method is optional. If provided, it
+          enables the calculation of "u-error". See `u_error`. Must have the same
+          signature as the PDF.
+
+    mode : float, optional
+        (Exact) Mode of the distribution. Default is ``None``.
+    center : float, optional
+        Approximate location of the mode or the mean of the distribution. This location
+        provides some information about the main part of the PDF and is used to avoid
+        numerical problems. Default is ``None``.
+    domain : list or tuple of length 2, optional
+        The support of the distribution. Must not contain `nan` values.
+        Default is ``None``. When ``None``:
+
+        * If a ``support`` method is provided by the distribution object
+          `dist`, it is used to set the domain of the distribution.
+        * Otherwise the support is assumed to be :math:`(-\infty, \infty)`.
+
+    order : int, optional
+        Order of the interpolating polynomial. Valid orders are between 3 and 17.
+        Higher orders result in fewer intervals for the approximations. Default
+        is 3.
+    u_resolution : float, optional
+        Set maximal tolerated u-error. Values of u_resolution must at least 1.e-15 and
+        1.e-5 at most. Notice that the resolution of most uniform random number sources
+        is 2-32= 2.3e-10. Thus a value of 1.e-10 leads to an inversion algorithm that
+        could be called exact. For most simulations slightly bigger values for the
+        maximal error are enough as well. Default is 1e-10.
+    max_intervals : int, optional
+        Set maximum number of intervals. max_ivs must be at least 100 and at most 1000000.
```

Comment:
maybe: The allowed range is [100, 1_000_000] (to get rid of `max_ivs`)

Reply:
Done.